app/main.py

```diff
@@ -1,3 +1,24 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+from pathlib import Path
+
+from app.customer import Customers
+from app.shops import Shops
+
+
+def shop_trip() -> None:
+    path_ = Path(__file__).parent.parent
+    path_new = path_.joinpath("app").joinpath("config.json")
+    with open(path_new, "r") as data_file:
+        config_file = json.load(data_file)
+        fuel_price = config_file["FUEL_PRICE"]
+
+        for shop in config_file["shops"]:
+            market = Shops(shop)
```

Comment:
This variable is redundant

Reply:
You can use list comprehension to store all shops in list and then pass them to choice_of_options method
Or you can remove the assignment and use class attribute which you define in Shops class.
For me, first option is better